src/components/PharmacyCard/PharmacyCard.tsx

```diff
@@ -25,13 +31,30 @@ interface PharmacyProps {
 export function PharmacyCard(props: PharmacyProps) {
   const [booking, setBooking] = useState(props.booking);
   const [lastUpdated, setlastUpdated] = useState(props.lastUpdated);
+  const [showMap, setShowMap] = useState(false);
 
   const updatePharmacy = (availability: boolean) => {
     const time = new Date();
     setlastUpdated(time.toLocaleString());
     setBooking(availability);
   };
 
+  const showTheMap = () => {
```

Comment:
Without the bool prefix, there could be some confusion reading code and seeing `showMap` and showTheMap` and what they are doing. They both start with verbs, so they both sound like function.

Reply:
Yeah, this should be an anon function instead of a named wrapper